alvr/session/src/settings.rs

```diff
@@ -1114,6 +1121,16 @@ TCP: Slower than UDP, but more stable. Pick this if you experience video or audi
 
     pub client_discovery: Switch<DiscoveryConfig>,
 
+    #[schema(strings(
+        help = r#"Wether ALVR should try to automatically launch the client when establishing a wired connection."#
+    ))]
+    pub client_autolaunch: bool,
+
+    #[schema(strings(
+        help = r#"Which type of client should ALVR look for when establishing a wired connection."#
+    ))]
+    pub client_flavor: ClientFlavor,
```

Comment:
I would rename these to `wired_client_autolaunch` and `wired_client_version_check`. I would also switch the order of these two settings. The reason is `wired_client_autolaunch` requires `wired_client_version_check`

Reply:
version_check makes no sense tho, this isn't about a the v version, which is probably going to confuse people

Reply:
Right. it's that "flavor" may sound a bit funny. I don't have better ideas so let's use `wired_client_flavor_check`

Reply:
wired_client_type and adjust the help string to "Which type of release of the client ..."